src/opencl_common.c

```diff
@@ -1260,7 +1259,7 @@ static char *include_source(char *pathname, int sequential_id, char *opts)
 	        opencl_driver_ver(sequential_id),
 	        opts ? opts : "");
 #if I_REALPATH
-	libc_free(full_path);
+	free(full_path);
 #else
 	MEM_FREE(full_path);
 #endif
```

Comment:
I believe 1261..1265 can all be unconditional MEM_FREE now, the libc_free was a memdbg thing.

Reply:
Ok.  I do not have an opencl build environment atm. that is why I tried to make no (or as few) code changes as possible to all opencl files.  I will remove the conditional (and let the build bots tell me if they are happy or not)